waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -671,9 +673,9 @@ proc resume*(ws: WakuStore, peerList: Option[seq[RemotePeerInfo]] = none(seq[Rem
           continue
         
       ws.messages.add(indexedWakuMsg)
-      waku_store_messages.inc(labelValues = ["stored"])
-      
       added = added + 1
+    
+    waku_store_messages.set(ws.messages.len.int64, labelValues = ["stored"])
```

Comment:
It may not be in the scope of the current PR, though, some of the messages may fail to be stored in the DB, therefore the number set in here would be inconsistent with the actual size of DB.

Reply:
Good point. This gauge is meant to track the total number of messages in _memory_. With the recently adopted store limits, we expect this to reach a maximum and remain constant - a bit boring to watch on a graph, but a good indication that our memory management works as expected. You're right though that we do not have a good way to keep track of messages in the DB, which won't be similarly limited. I'll see what I can add.